src/interactive-auth.js

```diff
@@ -138,7 +147,10 @@ InteractiveAuth.prototype = {
             // if we have no flows, try a request (we'll have
             // just a session ID in _data if resuming)
             if (!this._data.flows) {
-                this._doRequest(this._data);
+                this._setBusyCallback(true);
```

Comment:
Shouldn't the callback be optional? If so, we need to check for the callback everywhere it's used or else set it to a no-op function.

Reply:
oops - did it in one place but nowhere else